### PR TITLE
change snake_case run template yaml to camelCase

### PR DESCRIPTION
**Pull request by mukundansundar**
Signed-off-by: Mukundan Sundararajan <65565396+mukundansundar@users.noreply.github.com>

# Description

To be consistent with Dapr, snake_case in yaml run template is changed to camelCase.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1182 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by yaron2 — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

File: pkg/standalone/testdata/runfileconfig/test_run_config_empty_app_dir.yaml [hunk 1]
```diff
@@ -1,24 +1,24 @@
 version: 1
 common:
-  resources_path: ./app/resources
-  app_protocol: HTTP
-  app_health_probe_timeout: 10
+  resourcesPath: ./app/resources
+  appProtocol: HTTP
+  appHealthProbeTimeout: 10
   env:
     - name: DEBUG
```
Comment (mukundansundar) [inline pkg/standalone/testdata/runfileconfig/test_run_config_empty_app_dir.yaml:7]: @pravinpushkar please fix this unit test and remove these fields if not necessary. 
please fix in #1181

File: pkg/standalone/testdata/runfileconfig/test_run_config_invalid_path.yaml [hunk 2]
```diff
@@ -1,25 +1,25 @@
 version: 1
 common:
-  resources_path: ./app/resources
-  app_protocol: HTTP
-  app_health_probe_timeout: 10
+  resourcesPath: ./app/resources
+  appProtocol: HTTP
+  appHealthProbeTimeout: 10
   env:
     - name: DEBUG
```
Comment (mukundansundar) [inline pkg/standalone/testdata/runfileconfig/test_run_config_invalid_path.yaml:7]: same here

File: pkg/standalone/testdata/runfileconfig/test_run_config_empty_app_dir.yaml (hunk 1 above)
Comment (pravinpushkar) [reply pkg/standalone/testdata/runfileconfig/test_run_config_empty_app_dir.yaml:7]: addressed

File: pkg/standalone/testdata/runfileconfig/test_run_config_invalid_path.yaml (hunk 2 above)
Comment (pravinpushkar) [reply pkg/standalone/testdata/runfileconfig/test_run_config_invalid_path.yaml:7]: addressed

**Pull request merged**